scalableminds/webknossos
# Show fallback data for missing buckets

### Steps to test:
- copy a dataset and remove some cubes from the best mag level
- open a tracing
   - you should see at zoom step 2 that all data is there. 
   - If you zoom in, you should see that the data is missing
- go into the dataset settings and disable "Render Missing Data Black"
- reload the page
- if you zoom in, you should still see data (from a different zoom-step)

### Issues:
- fixes #3148

------
